### ISSUE TITLE
Hard to integrate with actions/create-release

### ISSUE THREAD

**Issue by Herve-M** [xresloader/upload-to-github-release#15]
Hello :-) 

I tried to use this action to upload release artifacts to an existing release created from [actions/create-release](https://github.com/actions/create-release) but looks like it don't play well :/ 

_Context:_ 
Use `actions/create-release` with: 
```
tag_name: ${{ github.ref }}
draft: true
prerelease: false
```
Actual behavior: The action don't find the created draft release, , creating one.
Waited behavior: The action should use the latest created draft release. 

Use `actions/create-release` with: 
```
tag_name: ${{ github.ref }}
draft: false
prerelease: false
```
Actual behavior: The action don't find the created draft release too, creating one.
Waited behavior: The action should use the latest created non-draft release. 

_Questions:_ 
* Would it be possible to have a option/input not to create a release if not found? The main responsability of this action is just to upload artifacts.
* Would it be possible to have a option/input to pass an release ID?

Thanks,
Herve-M

**Comment by owt5008137** [comment]
@Herve-M Thanks for your suggestion. ```actions/create-release``` will remove ```refs/tags/``` in ```tag_name``` field (The value of ```${{ github.ref }}``` is ```refs/tags/TAG_NAME``` when it's a tag commit, or ```refs/heads/BRANCH_NAME``` when it's a branch commit.) but [upload-to-github-release][1] will not. This is the reason [upload-to-github-release][1] could not find the existed releases.

>Questions:
>
>    Would it be possible to have a option/input not to create a release if not found? The main responsability of this action is just to upload artifacts.
>    Would it be possible to have a option/input to pass an release ID?

I will add ```release_id``` later.

[1]: https://github.com/xresloader/upload-to-github-release

**Comment by owt5008137** [comment]
```release_id``` is available on v1.3.0 or later. Example can be found here https://github.com/xresloader/upload-to-github-release-test/blob/master/.github/workflows/release.yml .
